enclave_wrangler/datasets.py

```diff
@@ -0,0 +1,355 @@
+#!/usr/bin/env python3
```

Comment:
I renamed the file by adding `.py` to the end. Crazy that it wasn't there before. But now can't see diff ;(